build.gradle

```diff
@@ -50,7 +50,7 @@ idea {
     }
 }
 dependencies {
-    compile("org.quartz-scheduler:quartz:2.2.3")
+    compile("org.quartz-scheduler:quartz:2.3.1")
```

Comment:
2.3.2 has been released recently. Maybe we could use it. WDYT?

Reply:
Quartz was updated to 2.3.2